crates/store/re_entity_db/src/entity_db.rs

```diff
@@ -442,24 +429,12 @@ impl EntityDb {
     fn on_store_deletions(&mut self, store_events: &[ChunkStoreEvent]) {
         re_tracing::profile_function!();
 
-        let Self {
-            data_source: _,
-            set_store_info: _,
-            last_modified_at: _,
-            latest_row_id: _,
-            entity_path_from_hash: _,
-            times_per_timeline,
-            tree,
-            data_store: _,
-            resolver: _,
-            query_caches,
-            stats: _,
-        } = self;
+        self.times_per_timeline.on_events(store_events);
+        self.query_caches.on_events(store_events);
+        self.time_histogram_per_timeline.on_events(store_events);
 
-        times_per_timeline.on_events(store_events);
-        query_caches.on_events(store_events);
-
-        tree.on_store_deletions(store_events);
+        // Tree deletions depend on data store, so data store must have been notified of deletions already.
```

Comment:
I'm confused by this comment -- the data store is the thing that emitted the changelog in the first place, how could it not know about what's in it?

Reply:
Yes, the problem here is that the store events are not actually being used by the tree anymore, so there's no direct dependency between them, which is inconsistent with the rest of the function, where everything else uses the store events. Just wanted to make that as clear as possible

Reply:
I guess I can just add the param back and then not use it